js/client.js

```diff
@@ -165,6 +166,9 @@
 					self.nameRegExp = new RegExp('(?:\\b|(?!\\w))' + escaped + '(?:\\b|\\B(?!\\w))', 'i');
 				}
 			});
+			this.on('change:settings', function () {
+				Dex.prefs('usersettings', self.get('settings'));
```

Comment:
This works? I thought `Storage.prefs` was the setter, and `Dex.prefs` was a compatibility-getter written for battles.

Reply:
Hm, apparently `Dex.prefs` forwards everything. It probably shouldn't.

Reply:
I just copied the pattern from the other code I saw. Should this be `Storage.prefs` instead?

Reply:
The old client is rather messy. For now, either probably works.

Reply:
Alright, well I went with `Storage.prefs`, but I dont think thats the more typical way given the code I was looking at.

Reply:
Actually, I went back on this. `Dex.prefs` for settings appears to be the overwhelming favorite, so Im going to opt for consistency.